### PR TITLE
feat: add an option to get all browser cookies

### PR DESCRIPTION
**Pull request by michellelievsay**
This change adds an option to return all browser cookies instead of only cookies with the domain of the loginUrl. We needed to add this capability to get our tests into a fully authenticated state.

## Description
We added a new option, options.getAllBrowserCookies, which, if true will cause GoogleSocialLogin to return all the cookies available at the time when getCookies() is called. We liked this approach because it allowed our tests to remain authenticated after the initial navigation, and additionally allows our tests to have the same set of cookies as a real user who authenticated through the UI.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

https://github.com/lirantal/cypress-social-logins/issues/7

## Motivation and Context

I'm working on a suite of cypress tests for a product that relies heavily on google authentication for most of its features. This plugin allowed us to set one cookie that we needed to get us authenticated initially, but we needed some additional cookies from other domains to keep our tests working in an authenticated state.

We initially looked at getting cookies for specific domains passed in as an array, but getting all cookies ended up working better for us and added less complexity.

## How Has This Been Tested?

We made the changes in node_modules to test. Tested with getAllBrowserCookies set to true, false, and without passing getAllBrowserCookies. All worked as expected and should not require any changes for folks currently using the plugin. 

## Screenshots (if appropriate):
Screen Shot 2019-09-17 at 1 33 34 PM

Screen Shot 2019-09-17 at 1 36 05 PM

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I added a picture of a cute animal cause it's fun

Piper :)
piper2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
   },
   "config": {
     "commitizen": {
-      "path": "./app/node_modules/cz-conventional-changelog"
```
Comment (michellelievsay) [inline package.json:116]: removed /app to allow us to commit using commitizen

Comment (lirantal) [reply package.json:116]: thanks!

**Review by lirantal — APPROVED** [review]
Looks great!

**Comment by lirantal** [comment]
@michellelievsay thanks for sending this, I'll go ahead and merge this in so we can release a new version that you can pull.

Would you be alright for sending another PR to update the README with the new option?

**Comment by michellelievsay** [comment]
@lirantal Oops forgot the readme! I'll update and have a new PR shortly.

**Pull request merged**